package.json

```diff
@@ -129,13 +129,39 @@
 							"description": "Store previously selected Quarkus extensions"
 						}
 					}
-        },
-        "quarkus.tools.symbols.showAsTree": {
-					"type": "boolean",
-					"default": "true",
-					"description": "Show Quarkus properties as tree",
-					"scope": "window"
-				}
+		        },
+		        "quarkus.tools.symbols.showAsTree": {
+						  "type": "boolean",
+						  "default": "true",
+						  "description": "Show Quarkus properties as tree",
+						  "scope": "window"
+		        },
+		        "quarkus.tools.validation.enabled": {
+						  "type": "boolean",
+						  "default": "true",
+						  "description": "Enabled validation",
+						  "scope": "window"
+				},
+		        "quarkus.tools.validation.unknown.severity": {
+						  "type": "string",
+		          "enum": [
+		            "none",
+		            "warning",
+		            "error"
+		          ],
+		          "default": "warning",
+		          "description": "Unknown Quarkus properties severity validation.",
+		          "scope": "window"
+		        },
+		        "quarkus.tools.validation.unknown.excluded": {
+		          "type": "array",
+		          "default": [],
+		          "items": {
+		            "type": "string"
+		          },
+		          "description": "Array of properties to ignore by Unknown Quarkus properties validation.",
```

Comment:
"Array of properties to ignore for unknown Quarkus properties validation."

Is there a reason why "Unknown" was capitalized?

Reply:
@xorye updated